lib/assert.js

```diff
@@ -81,8 +81,6 @@ x.throws = function (fn, err, msg) {
 				x.throws(function () {
 					if (fnErr instanceof Error) {
 						throw fnErr;
-					} else {
```

Comment:
Very well! I'll go ahead and amend my commit if you don't mind, and save myself the trouble of having to squash later.